src/modules/aws_codebuild/rpcs/import_source_credential.ts

```diff
@@ -0,0 +1,82 @@
+import { Build, CodeBuild, ImportSourceCredentialsInput } from '@aws-sdk/client-codebuild';
+import { ServerType } from '@aws-sdk/client-codebuild/dist-types/models/models_0';
+
+import { AWS, crudBuilderFormat } from '../../../services/aws_macros';
+import { Context, RpcBase, RpcResponseObject } from '../../interfaces';
+import { AwsCodebuildModule } from '../index';
+
+export class ImportSourceCredentialRpc extends RpcBase {
+  /** @internal */
+  module: AwsCodebuildModule;
+
+  /** @internal */
+  outputTable = {
+    arn: 'varchar',
+    status: 'varchar',
+    message: 'varchar',
+  } as const;
+
+  /** @internal */
+  importSourceCredentials = crudBuilderFormat<CodeBuild, 'importSourceCredentials', string | undefined>(
+    'importSourceCredentials',
+    input => input,
+    res => res?.arn,
+  );
+
+  validServerTypes = ['BITBUCKET', 'GITHUB', 'GITHUB_ENTERPRISE'];
+  validAuthTypes = ['BASIC_AUTH', 'OAUTH', 'PERSONAL_ACCESS_TOKEN'];
+
+  private makeError(message: string) {
+    return [
+      {
+        arn: '',
+        status: 'ERROR',
```

Comment:
we should unify at some point. I return OK/KO as there were other rpcs doing that. At some point we need to have a common standard for the returns of those functions

Reply:
Agreed. Let's raise it in the channel to discuss furtuer.